libservice/src/IpAddressNetlinkChecker.cpp

```diff
@@ -115,14 +116,21 @@ bool IpAddressNetlinkChecker::isV4AddressExternal(IPv4int addr) const {
 	return true;
 }
 
-static bool ipv6AddressContainsMappedIpv4Address(const in6_addr& addr) {
+bool IpAddressNetlinkChecker::ipv6AddressContainsMappedIpv4Address(const in6_addr& addr) const {
+	for (const auto& internalRange : {"::ffff:0:0:0/96", "64:ff9b::/96"}) {
+		if (isInRange(addr, internalRange)) {
+			return true;
+		}
+	}
+
```

Comment:
shouldn't lines 126 - 130 be replaced  with "return false" ?

Reply:
@pawsten These lines are used to detect this type of IP address correctly `::FFFF:192.168.0.5` (with dots)

Reply:
Ok, so we have 2 methods of checking address mask- written by you and old, You could remove old method  by adding  case "::ffff:0:0/96"  to line 120. I emphasise   that  case  "::ffff:0:0/96" is ~100% of cases and  check for "::ffff:0:0:0/96", "64:ff9b::/96"   is ~0% of addresses that are used nowadays ( they are rather historical and not almost not used today. So check for  ::ffff:0:0/96 should be first for performance reason